testlist/all.dat

```diff
@@ -30,3 +30,9 @@
 /x86_64
 
 /xtensa
+
+/renesas/rx65n/rx65n-grrose
+-rx65n-grrose:ipv6
```

Comment:
why remove ipv6 from build?

Reply:
Modified the defconfig file for ipv6 configuration. The testing of the modification is not done yet. Once the testing is completed, we will raise a PR to push the defconfig and then enable ipv6 build in precheck.